src/Base/Abstracts/Package.php

```diff
@@ -40,8 +40,15 @@
 use Pickle\Package\Util\Header;
 use Composer\Package\Version\VersionParser;
 
+/**
+ * Class Package
```

Comment:
What is this class really for? Okay, it is a class, yeah, but what does it generally do?